brain/python/CMakeLists.txt

```diff
@@ -58,6 +58,10 @@ set(_brain_python_libs Brain ${Boost_PYTHON${USE_BOOST_PYTHON_VERSION}_LIBRARY})
 # link python libraries.
 if(COMMON_LIBRARY_TYPE MATCHES "SHARED")
   list(APPEND _brain_python_libs ${PYTHON_LIBRARIES})
+else()
+  if(APPLE)
+    set_target_properties(brain_python PROPERTIES LINK_FLAGS "-undefined dynamic_lookup")
```

Comment:
comment what this does?

Reply:
Done. By the way, I also had to add an equivalent to --whole-archive